ops/jenkins/bfd-run-sonarqube/Jenkinsfile

```diff
@@ -65,9 +65,13 @@ spec:
         buildDiscarder(logRotator(numToKeepStr: '15', daysToKeepStr: '30'))
     }
 
-    // Run Every day at 22:00 PM Eastern / 03:00 AM UTC
+
     triggers {
-        cron(env.BRANCH_NAME == 'master' ? '00 03 * * 1-5' : '')
+        // UTC-time dictates that we look at Tue-Sat morning instead of Mon-Fri evening
+        // https://www.worldtimebuddy.com/?qm=1&lid=100,5128581&h=100&date=2024-12-17&sln=3-4&hf=1
+        // https://crontab.guru/#00_03_*_*_2-6
+        // At 03:00 on every day-of-week from Tuesday through Saturday.
+        cron(env.BRANCH_NAME == 'master' ? '00 03 * * 2-6' : '')
```

Comment:
Another target of opportunity. Jenkins runs in UTC. With the zero-offset, this was actually running Sunday evening through Thursday evening. Here, we run at 0300 UTC starting Tuesday (which is Monday evening in ET) through Saturday at 0300 (Friday evening in ET), making the historical Jenkins executions of this pipeline a little easier to follow as a result.